tests/devices/test_default_mixed.py

```diff
@@ -17,19 +17,53 @@
 
 import pytest
 import pennylane as qml
+
+from pennylane.ops import PauliZ, CZ, PauliX, Hadamard, CNOT, AmplitudeDamping, DepolarizingChannel
 from pennylane.devices.default_mixed import DefaultMixed
+from pennylane.wires import Wires
+
 import numpy as np
 
 INV_SQRT2 = 1 / np.sqrt(2)
 
+# Operations used for testing
```

Comment:
Would be nice to double-check that further supported (types of) operations work fine: e.g. `QubitUnitary`, `T`, single-qubit rotations, etc. From my perspective, this could serve the purpose of a sanity check for now (e.g. having `RX` as a single-qubit rotation, adding `Toffoli` as a three-qubit gate, etc.) with potentially having even further test cases for each operation at a later stage.

Reply:
Agree!